_posts/en/newsletters/2020-09-02-newsletter.md

```diff
@@ -0,0 +1,252 @@
+highlight a selection of the transcripts from the previous month.*
+
+- **nix-bitcoin**: nixbitcoindev appeared on the Stephan Livera Podcast to
+  discuss nix-bitcoin.  nix-bitcoin is an experimental but promising project
+  that aims to improve the security of installing Bitcoin and Lightning nodes. It
+  uses the functional Linux distribution NixOS where upgrading is equivalent to
+  installing from scratch due to atomicity. Through minimalism, reproducibility,
+  compartmentalization, and defense in depth, nix-bitcoin aims to reduce the attack
+  surface of a Bitcoin and Lightning software stack---ensuring programs are unaware of
+  other running processes unless they need to be. ([transcript][nixbitcoin
+  transcript], [audio][nixbitcoin audio])
+
+- **Taproot activation**: Eric Lombrozo, Luke Dashjr, and Aaron van Wirdum
+  discussed the alternative [taproot][topic taproot] activation proposals (see
+  [Newsletter #107][news107 taproot activation]) and gave their opinions in a
+  personal capacity on what lessons, if any, could be derived from activating the
+  segwit soft fork.  Lombrozo and Dashjr both believe that the taproot activation
+  process should not be drawn out and that all opposition, criticism, or review of
+  the proposed code changes should be completed prior to starting any possible
+  activation process. As a result, they support a single phase [BIP8][] activation
+  method with parameters still to be determined. Community feedback on activation
+  proposals continues to be collected. ([transcript][activation transcript],
+  [video][activation video])
+
+- **Signet**: Kalle Alm and AJ Towns participated in a discussion on
+  [signet][topic signet].  The design decisions of signet were explored as well
+  as the mechanics of how testnet and regtest work. For more details, see the
+  [_Default signet discussion_ news item](#default-signet-discussion).
+  ([transcript][signet transcript], [video][signet video])
+
+- **Bitcoin Core GUI meeting**: Anonymized participants, including designers and
+  developers, met to discuss the Bitcoin Core graphical user interface, its
+  current state, how it could be improved, and the constraints that
+  any possible revamp would be subject to. For example, one change that has been
+  discussed in the past is moving from Qt Widgets to the Qt QML framework which
+  is more flexible and easily customized. ([transcript][bitcoin core gui
+  transcript]).
+
+- **Sydney meetup discussion** Ruben Somsen and Rusty Russell participated in a
+  discussion on statechains, upgrading LN channels, and [lnprototest][]. Somsen
+  outlined how statechains (see [Newsletter #100][news100 statechains]) could
+  potentially be used to swap out your Lightning channel counterparty or, in the
+  case of DLCs (see [Newsletter #81][news81 dlc]), swap out a portion of your
+  position without closing the channel. With various possible Lightning channel
```

Comment:
nit: "possible" is a bit of a tic in this section, in that it is used in three paragraphs in a row in a way that is a bit needless, e.g. it could be omitted

Reply:
Also agree. I am trying to emphasize that none of these changes are inevitable but you're right it reads like a tic and if we are going to get rid of one "possible" better to take it from far future proposals like Lightning channel upgrades than proposals that are currently being discussed.

Reply:
I'd remove all three, though imo this one is maybe the one to keep if you keep any of them :p

Reply:
> I'd remove all three (ruthless editing ftw)

ACK.

Is there an `impossible Lightning channel`? I guess this is rather called a `broken Lightning channel`...